packages/grafana-ui/src/themes/GlobalStyles/fonts.ts

```diff
@@ -4,7 +4,7 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 export function getFontStyles(theme: GrafanaTheme2) {
   const grafanaPublicPath = typeof window !== 'undefined' && window.__grafana_public_path__;
-  const fontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
+  const fontRoot = grafanaPublicPath ? `${grafanaPublicPath}build/fonts/` : 'public/build/fonts/';
 
   return css([
```

Comment:
Could we change this to use webpack bundling imports instead?

```
import robotoMonoNormal from "./fonts/roboto-mono-normal.woff"
```

That would do the job as well, right?

Reply:
Yes nice, thanks :+1: Had to do some other tweaks to get it to compile properly/catch the types, but makes more sense now